JiraReporter/Program.cs

```diff
@@ -23,21 +23,24 @@ static void Main(string[] args)
             policy.SetCurrentOverride(options);
             policy.SetPermanentTaskLabel();
             policy.SetDraftMode(options);
+
             if (RunReport(policy, options) == true)
                 RunReportTool(args, policy, options);
+            else
+                Console.WriteLine("Unable to run report tool due to policy settings or final report already generated.");
         }
 
         private static bool RunReport(SourceControlLogReporter.Model.Policy policy, SourceControlLogReporter.Options options)
         {
-            var today = DateTime.Now.ToOriginalTimeZone().Date;
-            if (policy.IsDraft == false && today == policy.LastReportSentDate.Date)
-                return false;
-            if (policy.IsDraft == true && today == policy.LastReportSentDate.Date)
+            if (policy.LastReportSentDateFormat.Date.DayOfYear == DateTime.Now.ToOriginalTimeZone().Date.DayOfYear)
```

Comment:
Pentru ca in Policy ajungea sa fie DateTime, nu String si se vedea inuman.
 Acum se foloseste DayOfYear pentru ca altfel la DateTimeParse va da false intotdeauna din moment ce folosesti ToOriginalTimeZone si anumite proprietati nu mai corespundeai - verificarea ar trebui sa vada daca s-a realizat raportul in acea zi (verificare care se face strict pe asta)
Rezultatul final fiind verificarea corecta a datei (si a tipului), precum si salvarea datei mai 'frumos' in Policy.